### DISCUSSION TITLE
Add 1000 separator (,) when input value

### DISCUSSION THREAD

**Discussion by hoangduy1505** [firefly-iii/firefly-iii#5950] [General]
Hi,

I use VND currency so the value input have very number zero, can you add 1000 separator (,) when inputing value or when the user typing123 it will show 123 | 123,000 to choose for fast time typing
![Screenshot 2022-04-01 092744](https://user-images.githubusercontent.com/15534810/161183092-b543c5b1-0595-44b6-92d4-ac9ba32308a8.jpg)

**Comment by hoangduy1505** [comment]
You can see example
![1648788403173](https://user-images.githubusercontent.com/15534810/161197034-447b3dad-d2d2-4686-bc80-a172f73291b5.JPEG)
![1648788403218](https://user-images.githubusercontent.com/15534810/161197039-aadc9175-ed27-41ad-bafa-1314a583fcb4.JPEG)
![1648788403244](https://user-images.githubusercontent.com/15534810/161197040-6705e0e5-cbf7-4a08-9375-321d84e07566.JPEG)
![1648788403267](https://user-images.githubusercontent.com/15534810/161197042-c6a960ed-511a-4b15-b286-82354029d1e1.JPEG)
![1648788403301](https://user-images.githubusercontent.com/15534810/161197045-16e6d66a-fce4-46fc-8eb7-685a92b1517e.JPEG)

**Comment by JC5** [comment]
This is something that will be done in the new layout.

**Comment by JC5** [comment]
See also: https://github.com/firefly-iii/firefly-iii/issues/4770
